readthedocs/core/unresolver.py

```diff
@@ -247,6 +262,9 @@ def _unresolve_path(self, parent_project, path, check_subprojects=True):
         :param check_subprojects: If we should check for subprojects,
          this is used to call this function recursively when
          resolving the path from a subproject (we don't support subprojects of subprojects).
+        :param external_version_slug: Slug of the external version.
+         Used as the default version for single version projects
```

Comment:
```suggestion
         Used instead the default version for single version projects
```

It's not used _as_ but _instead_.